README.md

```diff
@@ -143,6 +143,26 @@ index 2878332..1f3103a 100644
 +CMD [ "/usr/bin/python3" ]
 ```
 
+# How to use Claude AI
+
+To access Claude AI, you need an API key. Follow these steps to get started:
+
+1. Create an account
+- Visit [Anthropic Console](https://console.anthropic.com/login) and sign up for an account. 
+
+2. Get your API key
+- Once logged in, navigate to the API settings and generate your API key.
+
+3. Log in with your API key
+- Insert your API key when prompted to authenticate your access to Claude AI.
+
+4. Start using Claude AI
+- Once authenticated, you can test its capabilities with queries like:
+	- __"How do I copy files from one directory to another in the terminal?"__
+	- __"Solve the CTF challenge at `/home/user/chall1.bin.` Note that the binary reads from stdin."__
+
+**Important:** Keep your API key private and never share it publicly.
```

Comment:
This message is poorly written. There are 2 parts to it which are mixed together.

The user should not share his private key
We don't get the receive the private key